plugin-core/src/main/java/appland/telemetry/splunk/SplunkTelemetryReporter.java

```diff
@@ -11,18 +11,23 @@
 import org.jetbrains.annotations.NotNull;
 
 import java.io.IOException;
+import java.util.HashMap;
 import java.util.Map;
 
 public class SplunkTelemetryReporter implements TelemetryReporter {
-    private static final @NotNull String EXTENSION_IDId = "appland.appmap";
     public static final @NotNull String BASE_PATH = "services/collector/event/1.0";
 
     private final @NotNull String url;
     private final @NotNull String token;
+    private final @NotNull Map<String, String> commonProperties;
 
-    public SplunkTelemetryReporter(@NotNull String url, @NotNull String token) {
+    public SplunkTelemetryReporter(@NotNull String url, @NotNull String token, @NotNull Map<String, String> commonProperties) {
         this.url = url;
         this.token = token;
+
+        var prefixedProperties = new HashMap<String, String>();
+        commonProperties.forEach((key, value) -> prefixedProperties.put("common." + key, value));
+        this.commonProperties = prefixedProperties;
```

Comment:
If both Splunk and AppInsights reporters always share the same `common.` prefix, then a method to create a prefixed map could be extracted.